Sources/Sentry/SentryTraceContext.m

```diff
@@ -112,20 +112,25 @@ - (SentryBaggage *)toBaggage
     NSMutableDictionary *result =
         @{ @"trace_id" : _traceId.sentryIdString, @"public_key" : _publicKey }.mutableCopy;
 
-    if (_releaseName != nil)
+    if (_releaseName != nil) {
```

Comment:
`l`: It would be nice to have such changes in a separate PR. It makes reviewing this easier.